Documentation/contributor-guide/roadmap.md

```diff
@@ -0,0 +1,47 @@
+# roadmap
+
+etcd uses GitHub milestones to track all tasks in each major or minor release. The roadmap.md file only records the most
+important tasks for each release.
+Please also refer to [here](https://docs.google.com/spreadsheets/d/1hFpIbkI0IDum0SPy8txj7bVHxQRrAB_82f1Yqz_Pa8M/edit#gid=0&fvid=1685003197)
```

Comment:
I think having too many sources will lead to many of them being unmaintained. The spreadsheet was good for creating the list, but I would not mention it to avoid confusion whether it or this file is more up to date.

Reply:
Yes, I had the same thought. But the linked spreadsheet has more context (e.g. issues/PRs/docs links), it's super useful when you want to quickly jump to the related issues or PRs for a specific roadmap item. Both the roadmap and the spreadsheet should be relatively stable. I suggest to keep it as it's for now, and we can revisit it (e.g. remove the link or get the simplified context included in this roadmap) when we see any issue. WDYT?

Reply:
Don't understand the advantage of sheet. You can add link to issues here.

Reply:
It isn't a big problem to me, but updated it since you insist on it. Lost some context (e.g. doc links).